### ISSUE TITLE
Getting exception printed when apigee lint command runs

### ISSUE THREAD

**Issue by sithik4git** [apigee/apigeelint#257]
Getting below error printed during apigee lint command runs in a pipeline Jenkins stage.
Apigee Lint report is getting generated successfully as expected but it is not coming out of the Jenkins stage successfully.
Jenkins is not giving proper error message to know the reason.
Suspecting the error message "exception: TypeError: t.value.endsWith is not a function" is the cause here.

Note: When I run the same apigeelint command for sharedflows, I am not seeing this exception and npm package warnings.
Console is very neat.

01:42:45  + **cd src/gateway**
01:42:45  + **pwd**
01:42:45  /var/lib/jenkins/workspace/DAAS-DPLUS/GATEWAY-TEAM/Abu-Apigeelint-Deploy-Apigee-Artifacts/src/gateway
01:42:45  + **npm install -g apigeelint**
01:42:46  npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
01:42:55  npm WARN deprecated core-js@2.6.12: core-js@<3.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.
01:42:55  npm WARN deprecated har-validator@5.1.5: this library is no longer supported
01:43:01  /usr/bin/apigeelint -> /usr/lib/node_modules/apigeelint/cli.js
01:43:01  
01:43:01  > core-js@2.6.12 postinstall /usr/lib/node_modules/apigeelint/node_modules/core-js
01:43:01  > node -e "try{require('./postinstall')}catch(e){}"
01:43:01  
01:43:01  + apigeelint@2.5.0
01:43:01  added 347 packages from 244 contributors in 16.595s
01:43:01  + **apigeelint -s my-proxy/apiproxy -f html.js**
01:43:02  exception: TypeError: t.value.endsWith is not a function
01:43:02  exception: TypeError: t.value.endsWith is not a function
01:43:02  exception: TypeError: t.value.endsWith is not a function
01:43:02  exception: TypeError: t.value.endsWith is not a function
01:43:02  exception: TypeError: t.value.endsWith is not a function
01:43:02  exception: TypeError: t.value.endsWith is not a function
01:43:02  exception: TypeError: t.value.endsWith is not a function
01:43:02  exception: TypeError: t.value.endsWith is not a function
01:43:02  exception: TypeError: t.value.endsWith is not a function
01:43:02  exception: TypeError: t.value.endsWith is not a function
01:43:02  exception: TypeError: t.value.endsWith is not a function
01:43:02  exception: TypeError: t.value.endsWith is not a function
01:43:02  exception: TypeError: t.value.endsWith is not a function
01:43:02  <!DOCTYPE html>
01:43:02      <head>
01:43:02          <title>Apigee Bundle Linter Report</title>

**Comment by kurtkanaskie** [comment]
Hmmm, just saw this today with 2.5.0
`TypeError: t.value.endsWith is not a function`
Seems the culprit is in CC005-unterminatedStrings.js for this change, if I revert to the commented all seems fine.
```
        // let invalid = tokens.filter( t => t.valid == false);
        // invalid.forEach( t => {
        //   condition.addMessage({
        //     source: condition.getExpression(),
        //     line: condition.getElement().lineNumber,
        //     column: condition.getElement().columnNumber,
        //     plugin,
        //     message: `invalid token. (${t.value})`
        //   });
        //   flagged = true;
        // });

        let constants = tokens.filter( t => t.type == 'constant');
        constants.forEach( t => {
          if (t.value &&
              ((t.value.endsWith('"') && !t.value.startsWith('"')) ||
               ( !t.value.endsWith('"') && t.value.startsWith('"')) )
             ) {
          condition.addMessage({
            source: condition.getExpression(),
            line: condition.getElement().lineNumber,
            column: condition.getElement().columnNumber,
            plugin,
            message: `Possible unterminated string: (${t.value})`
          });
          flagged = true;
          }
        });
```

**Comment by ssvaidyanathan** [comment]
Fixed in v2.5.1
